runtime/opt/taupage/init.d/02-start-berry.sh

```diff
@@ -1,6 +1,14 @@
 #!/bin/bash
 
 # only start berry service if "mint_bucket" was defined
-grep 'mint_bucket' /meta/taupage.yaml && service berry start
-
+grep 'mint_bucket' /meta/taupage.yaml
+if [ $? -eq 0 ]; then
+    # start berry once to make sure to have valid credentials for the rest of taupage init
+    berry --once
```

Comment:
this command line is missing the path to the credentials, compare to https://github.com/zalando-stups/taupage/blob/master/runtime/etc/init/berry.conf#L8